### PR TITLE
Add references to the Python client & Python examples to various places

### PR DESCRIPTION
**Pull request by fnesveda**
I've added a bunch of Python client references to places where only the JS client was referenced, and added some Python examples to relevant places.

Python tutorials will come soon in a separate PR.

### CODE REVIEW

File: docs/actors/development/base_docker_images.md [hunk 1]
```diff
@@ -18,6 +26,8 @@ Apify provides the following Docker images that can be used as a base for user a
 
 For a full list of available images, [see the Apify SDK Docker image guide](https://sdk.apify.com/docs/guides/docker-images). Note that some images available in the Apify UI can be marked as deprecated. This means that they should no longer be used for new projects and old projects are encouraged to migrate to one of the non-deprecated images.
 
-All images come in two versions: the **latest** tag corresponds to the stable version and **beta** to images where we test new features. Use the beta version at your own risk.
+## [](#python-actor-images) Images with Apify Client for Python preinstalled
 
-Note that all Apify Docker images are pre-cached on Apify servers in order to speed up the actor builds and runs. The source code used to generate the images is available in the [apify-actor-docker](https://github.com/apifytech/apify-actor-docker) GitHub repository.
+These images contain the [Apify API client for Python](https://docs.apify.com/apify-client-python) preinstalled.
```
Comment (m-murasovs) [inline docs/actors/development/base_docker_images.md:31]: same rephrase as above, please

File: docs/actors/development/source_code.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ For more information about Dockerfile syntax and commands, see the [Dockerfile r
 
 Note that `apify/actor-node-basic` is a base Docker image provided by Apify. There are other base images with other features available. However, you can use arbitrary Docker images as the base for your actors, although using the Apify images has some performance advantages. See [base Docker images]({{@link actors/development/base_docker_images.md}}) for details.
 
-By default, all Apify base Docker images start your Node.js application same way as **npm start** does, i.e. by running the command specified in the **package.json** file under the **scripts** - **start** key. The default **package.json** file is similar to the following.
+By default, all Apify base Docker images with the Apify SDK start your Node.js application same way as **npm start** does, i.e. by running the command specified in the **package.json** file under the **scripts** - **start** key. The default **package.json** file is similar to the following.
```
Comment (m-murasovs) [inline docs/actors/development/source_code.md:103]: I know it wasn't your change, but please update to `same way` => `the same way`

File: docs/api_v2/api_v2_reference.apib [hunk 3]
```diff
@@ -368,8 +370,9 @@ and it can be described using the following pseudo-code:
 If all requests sent by the client implement the above steps,
 the client will automatically use the maximum available bandwidth for its requests.
 
-Note that the <a href="https://docs.apify.com/api/apify-client-js/latest">apify-client</a> NPM package
-uses the exponential backoff algorithm transparently, so that you do not need to worry about it.
+Note that the Apify API clients <a href="https://docs.apify.com/api/apify-client-js/latest">for Javascript</a>
```
[the remark this answers is not in the document]
Comment (m-murasovs) [reply docs/api_v2/api_v2_reference.apib:373]: Not sure why this was using a HTML anchor while the others are using the Markdown. Could you update these links to use the Markdown links?

File: docs/storage/key_value_store.md [hunk 4]
```diff
@@ -199,6 +214,12 @@ In the [JavaScript API client](/apify-client-js), you can access a store using [
 const otherStoreClient = apifyClient.keyValueStore('jane-doe/old-store');
 ```
 
+The same works in the [Python API client](/apify-client-python), you can access a store using [its client](/apify-client-python#keyvaluestoreclient) again.
```
Comment (m-murasovs) [inline docs/storage/key_value_store.md:217]: Same comment here as on the dataset page

File: docs/tutorials/integrations.md [hunk 5]
```diff
@@ -20,7 +20,8 @@ Integrations use [APIs](https://www.smashingmagazine.com/2018/01/understanding-u
 Our [RESTful API](/api/v2#) allows you to control the Apify platform from any application.
 You can create [actors](/api/v2#/reference/actors/actor-collection/create-actor) and [tasks](/api/v2#/reference/actor-tasks/task-collection/create-task),
 [start and stop your runs](/api/v2#/reference/actor-tasks/run-task-synchronously/run-task-synchronously-(post)),
-and [manage your data](/api/v2#/reference/datasets/item-collection/put-items) using only HTTP requests (or our handy [API client]({{@link apify_client_js.md}})). Meanwhile, webhooks allow you to perform tasks like sending HTTP requests or notification on when certain [events]({{@link webhooks/events.md}}) occur.
+and [manage your data](/api/v2#/reference/datasets/item-collection/put-items) using only HTTP requests (or our handy API clients [for Javascript]({{@link apify_client_js.md}}) and [for Python]({{@link apify_client_python.md}})).
+Meanwhile, webhooks allow you to perform tasks like sending HTTP requests or notification on when certain [events]({{@link webhooks/events.md}}) occur.
```
Comment (m-murasovs) [inline docs/tutorials/integrations.md:24]: `or notification on when` => `or notifications when`

Is this the typo you mentioned? 😄

File: docs/tutorials/integrations.md [hunk 6]
```diff
@@ -20,7 +20,8 @@ Integrations use [APIs](https://www.smashingmagazine.com/2018/01/understanding-u
 Our [RESTful API](/api/v2#) allows you to control the Apify platform from any application.
 You can create [actors](/api/v2#/reference/actors/actor-collection/create-actor) and [tasks](/api/v2#/reference/actor-tasks/task-collection/create-task),
 [start and stop your runs](/api/v2#/reference/actor-tasks/run-task-synchronously/run-task-synchronously-(post)),
-and [manage your data](/api/v2#/reference/datasets/item-collection/put-items) using only HTTP requests (or our handy [API client]({{@link apify_client_js.md}})). Meanwhile, webhooks allow you to perform tasks like sending HTTP requests or notification on when certain [events]({{@link webhooks/events.md}}) occur.
+and [manage your data](/api/v2#/reference/datasets/item-collection/put-items) using only HTTP requests (or our handy API clients [for Javascript]({{@link apify_client_js.md}}) and [for Python]({{@link apify_client_python.md}})).
```
Comment (m-murasovs) [inline docs/tutorials/integrations.md:23]: `Javascript` => `JavaScript`

File: docs/tutorials/integrations.md (hunk 5 above)
Comment (fnesveda) [reply docs/tutorials/integrations.md:24]: Nah, there was a missing and, I already fixed that one 😄

**Review by m-murasovs — APPROVED** [review]

File: docs/actors/development/base_docker_images.md [hunk 7]
```diff
@@ -8,7 +8,15 @@ paths:
 
 # [](#base-docker-images) Base Docker images
 
-Apify provides the following Docker images that can be used as a base for user actors. You can read more about them in the [Apify SDK Docker image guide](https://sdk.apify.com/docs/guides/docker-images).
+Apify provides several Docker images that can be used as a base for user actors.
+
+All images come in two versions: the **latest** tag corresponds to the stable version and **beta** to images where we test new features. Use the beta version at your own risk.
+
+Note that all Apify Docker images are pre-cached on Apify servers in order to speed up the actor builds and runs. The source code used to generate the images is available in the [apify-actor-docker](https://github.com/apifytech/apify-actor-docker) GitHub repository.
```
Comment (mtrunkat) [inline docs/actors/development/base_docker_images.md:15]: Not true ATM but we will fix this ASAP hopefully

**Review by mtrunkat — APPROVED** [review]

File: docs/actors/development/base_docker_images.md (hunk 7 above)
Comment (fnesveda) [reply docs/actors/development/base_docker_images.md:15]: Yeah, this was just copied over from the old texts.

**Pull request merged**